CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## unreleased
+* Change JSON serializer to use gson
```

Comment:
Let's also note the specific breaking change we know about, the error changing from `JsonParseException` to `MalformedJsonException`